pkg/sync/closer.go

```diff
@@ -95,8 +95,9 @@ func (c *Closer) OnClose(handler func()) bool {
 
 // OnCloseAlways registers the passed callback to be called when the Closer is
 // closed.
-// If the Closer is already closed, immediately executes the callback. Returns
-// whether the closer was not yet closed.
+// If the Closer is already closed, it executes the callback immediately and
+// returns false.
+// If the Closer is not yet closed and call back was registered, returns true.
```

Comment:
Same comment as above.

Reply:
Updated.

Reply:
see discussion on line 83 above

Reply:
In `OnCloseAlways`, previously it said 
```
If the Closer is already closed, immediately executes the callback
```
It was not clear whether the callback was registered or not in the first read. In the proposed revision, I tried to make it more explicit. But, if you feel, the previous version is already clear, I wouldn't mind dropping this change.